ajax/ajax.md

```diff
@@ -66,7 +67,7 @@ Before AJAX, the request/response cycle could only happen with the web page *as
 AJAX, however, enabled the browser to make requests and only re-render *parts* of a webpage. This was beneficial performance-wise, because smaller strings of HTML, rather than all the HTML for the entire page, was all that was needed to be sent by the server and rendered by the browser. This also made the end user's browsing experience smoother: interactions with the webpage resulted in changes on screen without being interrupted by a reload.
 
 ### Remember
-[Thread of Execution](../javascript/javascript-9.md)
+[Thread of Execution](/javascript/javascript-async-9)
```

Comment:
Add file extension.

Reply:
Okay

Reply:
what about the slides access ?

Reply:
Yeah, add them back. As confusion would arise.

Reply:
@alodahl Please give him slide access.